Cargo.toml

```diff
@@ -50,12 +50,12 @@ default = [
 ]
 
 [dependencies]
-iced_style = { git = "https://github.com/hecrj/iced.git", rev = "adce9e0" }
+iced_style = { git = "https://github.com/iced-rs/iced.git", rev = "adce9e0" }
```

Comment:
This change breaks every example (at least for me). This change would need to be taken on every example as well 
Would you like to open up another PR with this changes and remove this change from this PR for now?

Reply:
yeah id request that he removes this change from his PR so we can go fix these in a separate PR.